src/openrct2/scripting/ScriptEngine.h

```diff
@@ -47,7 +47,7 @@ namespace OpenRCT2
 
 namespace OpenRCT2::Scripting
 {
-    static constexpr int32_t OPENRCT2_PLUGIN_API_VERSION = 88;
+    static constexpr int32_t OPENRCT2_PLUGIN_API_VERSION = 89;
```

Comment:
Develop branch currently has API 89, this needs to be rebased and updated to 90.